lib/resolvers/project-version.js

```diff
@@ -51,8 +35,27 @@ module.exports = ({ models }) => ({ action, data, id }, transaction) => {
       });
   }
 
-  if (action === 'update' || action === 'patch') {
-    return updateVersion()
+  if (action === 'patch') {
```

Comment:
Are we _absolutely_ sure there's no lingering `update`s?

Reply:
I would be surprised as I'm 99% sure they autoresolved, however we could leave it to be safe?